src/vos/vos_internal.h

```diff
@@ -1286,6 +1286,12 @@ vos_offload_exec(int (*func)(void *), void *arg)
 		return func(arg);
 }
 
+static inline bool
+umoff_is_null(umem_off_t umoff)
+{
+	return umoff == UMOFF_NULL;
```

Comment:
move to mem.h ?

Reply:
Yes, can be moved to the comment header.